testcases/PowerNVDump.py

```diff
@@ -791,12 +791,49 @@ def runTest(self):
                 boot_type = self.kernel_crash(crash_type="hmc")
                 self.verify_dump_file(boot_type)
 
+class KernelCrash_KdumpWorkLoad(PowerNVDump):
+
+    # This test verifies kdump/fadump after running ebizzy.
+    # ebizzy url needs to be given in ~/.op-test-framework.conf.
+    # Ex: url=http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3/ebizzy-0.3.tar.gz
+
+
+    def runTest(self):
+        if not self.url:
+            raise self.skipTest("Provide ebizzy url in op-test-framework.conf")
+        self.setup_test()
+        if self.distro == "rhel":
+            self.c.run_command("yum -y install make gcc wget; cd /tmp", timeout=120)
+        elif self.distro == "ubuntu":
+            self.c.run_command("apt-get install -y make gcc wget; cd /tmp", timeout=120)
+        else:
+            self.c.run_command("zypper install -y make gcc wget; cd /tmp", timeout=120)
```

Comment:
Not that anything is wrong with the above code but may be it can be organised in a better way as follows:
```
         if self.distro == "rhel":
               cmd = "yum -y install make gcc wget"
         elif self.distro == "ubuntu":
             cmd = "apt-get install -y make gcc wget"
         else:
             cmd = "zypper install -y make gcc wget"

          self.c.run_command(cmd, timeout=120)
          self.c.run_command("cd /tmp", timeout=120)
```